resources/views/tasks.blade.php

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>{{ $title }}</title>
+        <link rel="stylesheet" href="{{ mix('/css/app.css') }}">
```

Comment:
Nghiên cứu http://laravel-recipes.com/recipes/184/generating-a-link-to-a-css-file để fix chỗ này em nhé